Project/src/Utils/Utils.js

```diff
@@ -64,6 +101,24 @@ export const utils = {
             return 'Unknown Identifier';
         }
     },
+    constructIdentifierFromStringMri: (mri) => {
+        if (mri.startsWith("8:orgid")) {

```

Comment:
can we remove all this now the `@azure/communication-common` has helper functions like `createIdentifierFromRawId` for this